experiments/openebs/openebs-nfs-provisioner-kill/openebs_nfs_provisioner_kill_k8s_job.yml

```diff
@@ -0,0 +1,74 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: openebs-nfs-provisioner-kill-
+spec:
+  template:
+    metadata:
+      labels:
+        experiment: openebs-nfs-provisioner-kill
+    spec:
+      serviceAccountName: %CHAOS_SERVICE_ACCOUNT%
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: litmuschaos/ansible-runner:ci
+        imagePullPolicy: Always
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+           
+          #provide openebs namespace
+          - name: NFS_PROVISIONER_NAMESPACE
+            value: ''
+
+          # provide application namespace
+          - name: NFS_PROVISIONER_LABEL
+            value: ''
+
+          # provide application labels
+          - name: NFS_PVC
+            value: ''
+
+          - name: NFC_PVC_NS
```

Comment:
NFC_PVC_NS can be different from NFS_PROVISIONER_NAMESPACE?

Reply:
NFC_PVC_NS is unused, removing it